examples/MagicWeather/README.md

```diff
@@ -15,13 +15,84 @@ See minimum react-native version requirements for RevenueCat's *Purchases* SDK [
 
 ## Features
 
-This sample demonstrates:
+| Feature                          | Sample Project Location                   |
+| -------------------------------- | ----------------------------------------- |
+| 🕹 Configuring the *Purchases* SDK  | [App.js](App.js#L17) |
+| 💰 Building a basic paywall         | [src/screens/PaywallScreen/index.js](src/screens/PaywallScreen/index.js) |
+| 🔐 Checking subscription status     | [src/screens/WeatherScreen/index.js](src/screens/WeatherScreen/index.js#L30) |
```

Comment:
Now that I see this, I worry we will forget to update this lines. I think we should probably just point to `WeatherScreen /index.js`?

Reply:
```suggestion
| 🔐 Checking subscription status     | [src/screens/WeatherScreen/index.js](src/screens/WeatherScreen/index.js) |
```